### PR TITLE
Fixed inventory alignment. Issue #2158

### PR DESCRIPTION
**Pull request by marcgenesis**
Hello, this is my attempt at fixing the inventory alignment. Issue #2158.

First time using git/github, first time working in jade. Hope it's fine! Please let me know if I could have done something better!

I also changed the text in character.json but I don't think the file is tracked in git... or I'm not sure where to find the node_modules folder (which branch to checkout). If there is a way of doing it any help would be appreciated.

Thanks.

### CODE REVIEW

**Comment by SabreCat** [comment]
Ooh! Thanks for taking this on @marcgenesis, I'd love to see that straightened out.

character.json and many other files not specific to the website front-end can be found in the [habitrpg-shared](https://github.com/HabitRPG/habitrpg-shared) repository. This allows us to make updates that can be applied simultaneously across the site, the mobile app, etc., keeping text, images, and so forth consistent.

File: views/options/inventory/inventory.jade [hunk 1]
```diff
@@ -2,16 +2,17 @@ script(type='text/ng-template', id='partials/options.inventory.equipment.html')
   .row-fluid
     .span6.border-right
       h3=env.t('battleGear')
```
Comment (SabreCat) [inline views/options/inventory/inventory.jade]: Did you mean for this to have `(style="display: inline-block; vertical-align: middle;")` as well? Without that, the question bubble falls to the next line.

**Comment by SabreCat** [comment]
Travis build fail looks unrelated. Assuming a correction on the annotated line, this'll be quick to merge.

@marcgenesis, what's your Habit User ID so we can zing you a contrib level when this merges?

File: views/options/inventory/inventory.jade (hunk 1 above)
Comment (paglias) [inline views/options/inventory/inventory.jade]: If it's possible could you move the css to a file in public/css, we should avoid inline css

Comment (marcgenesis) [inline views/options/inventory/inventory.jade]: @SabreCat Yes. I thought I did it but being new to the whole git/jade/node scene I might have messed it up somehow! It's all fixed now.

@paglias I agree and done. I added a class to inventory.styl. Let me know if I should choose a name more generic.

**Comment by marcgenesis** [comment]
I have modified character.json in habitrpg-shared accordingly. Please see https://github.com/HabitRPG/habitrpg-shared/pull/124

My Habit User ID is f3bc103a-fbcd-481a-9734-41e7520cb994 . Thanks!

**Comment by lefnire** [comment]
Cool beans!
- @paglias heads up re: bs3 merge
- @SabreCat you get @marcgenesis contrib.lvl++ already?
- @SabreCat @marcgenesis Let's start moving towards underlined text for popover help contexts, instead of (?) ([see example code](https://github.com/HabitRPG/habitrpg/blob/develop/views/static/plans.jade#L34)). I think it'll help remove some clutter. Nothing at all urgent, just heads up

**Comment by paglias** [comment]
@lefnire merged, @marcgenesis it would be better to avoid inline css

**Comment by SabreCat** [comment]
@marcgenesis: Contrib level awarded!

@lefnire: Good call. I'd been running with the existing help-bubble model, but with the more complex elements introduced in Classes throwing ? icons everywhere, I can totally see doing underlines instead. Want me on that as a project?

**Comment by marcgenesis** [comment]
@paglias I just saw that I forgot one of the inline style. I will fix it.

@lefnire and @SabreCat, I can look into replacing the current ? with underline hints instead.

**Comment by lefnire** [comment]
@marcgenesis cool man, thanks!

**Pull request merged**